tests/php/includes/class-wc-tracker-test.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Unit tests for the WC_Tracker class.
+ *
+ * @package WooCommerce\Tests\WC_Tracker.
+ */
+
+/**
+ * Class WC_Tracker_Test
+ */
+class WC_Tracker_Test extends \WC_Unit_Test_Case {
+
+	/**
+	 * Utility method to add filter on woocommerce_admin_disabled
+	 */
+	public function __disable_woocommerce_admin() {
+		return true;
+	}
+
+	/**
+	 * Test the tracking of wc_admin being disabled via filter.
+	 */
+	public function test_wc_admin_disabled_get_tracking_data() {
+		$tracking_data = WC_Tracker::get_tracking_data();
```

Comment:
You can replace with this to avoid making `get_tracking_data` public:

```suggestion
		$posted_data = null;

		add_filter(
			'pre_http_request',
			function( $false, $args, $url ) use ( &$posted_data ) {
				$posted_data = $args;
				return true;
			}, 3, 10
		);

		WC_Tracker::send_tracking_data( true );
		$tracking_data = json_decode( $posted_data['body'], true );
```